include/pm/device.h

```diff
@@ -361,6 +361,43 @@ void pm_device_children_action_run(const struct device *dev,
 		pm_device_action_failed_cb_t failure_cb);
 
 #if defined(CONFIG_PM_DEVICE) || defined(__DOXYGEN__)
+/**
+ * @brief Initialize a device state to #PM_DEVICE_STATE_SUSPENDED.
+ *
+ * By default device state is initialized to #PM_DEVICE_STATE_ACTIVE. In
+ * general, this makes sense because the device initialization function will
+ * resume and configure a device, leaving it operational. However, when device
+ * runtime PM is enabled, the device may not be resumed and the init function
+ * will just enable device runtime PM. If that is the case, this function can be
+ * used to set the initial device state to #PM_DEVICE_STATE_SUSPENDED.
```

Comment:
Documentation mentions runtime PM as usecase. It should be updated to be more generic or explain other scenarios where using this API can be useful.

Reply:
Explicit references to runtime PM have been removed.